_includes/v23.1/ui/insights.md

```diff
@@ -0,0 +1,221 @@
+## Workload Insights tab
```

Comment:
this page appears to be the same across v22.2 and v23.1?  at least I couldn't discern any big differences.  If they are the same, see my suggestions on v22.2 version of this page - from looking through, all the same suggestions would apply (and please feel free to ignore if I'm missing some different details that would make them not apply of course!)

Reply:
ah ok I took a closer look.  at least from what I can see there are 3 areas of difference, but they don't really change my suggestions which are basically to add as many links to things as possible :-)